cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/aws/Attribute.java

```diff
@@ -29,7 +29,7 @@
         this.value = value;
     }
 
-    public String getValue() {
+    private String getValue() {
```

Comment:
this enum also has a method named getAttribute, this method has the same functionality as the getValue methods from the other enums. This should be changed for consistency.